### ISSUE TITLE
Backend: create proof at Lissi API

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#144]
## User story

1. As a {HR-employee}
2. I want / need {to create a new proof which I can use to prove credentials}
3. So that {I can request information from DI (via credential)}

## Acceptance criteria
* backend sends request to Lissi API to create a proof with data (which is later received from frontend)
* response from Lissi API is successfully received

## Definition of done (DoD)

- Added only after week 5
- The same for all features
- Here goes the project specific part

## DoD general criteria

- Feature has been fully implemented
- Feature has been merged into the mainline
- All acceptance criteria were met
- Product owner approved features
- All tests are passing
- Developers agreed to release

**Comment by jackDS008** [comment]
- [ ] Implementierung
- [ ] Tests
- [ ] OpenAPI

**Comment by annikakrause** [comment]
Is already implemented because this issue wasn't tagged in #167 and @valentinBraeutigam and I thought, that #167 includes also the backend (we read "final button below all other things to create proof" as it should work completely)
